lighthouse-core/audits/audit.js

```diff
@@ -94,6 +101,42 @@ class Audit {
     };
   }
 
+  /**
+   * @param {!Audit} audit
+   * @param {!AuditResult} result
+   * @return {{score: number, scoreDisplayMode: string, informative: boolean, rawValue: boolean}}
+   */
+  static _normalizeAuditScore(audit, result) {
+    let score = typeof result.score === 'undefined' ? result.rawValue : result.score;
+    let informative;
+    let rawValue;
+
+    if (typeof score === 'boolean' || score === null) {
+      score = score ? 1 : 0;
+    } else {
+      score = clampTo2Decimals(score);
+    }
+
+    // If the audit was determined to not apply to the page, we'll reset it as informative only
+    if (result.notApplicable) {
+      score = 1;
+      rawValue = true;
+      informative = true;
+    }
+
+    if (!Number.isFinite(score)) throw new Error(`Invalid score: ${score}`);
+    if (score > 1) throw new Error(`Audit score for ${audit.meta.name} is > 1`);
+
+    const scoreDisplayMode = audit.meta.scoreDisplayMode || Audit.SCORING_MODES.BINARY;
+
+    return {
+      score,
+      scoreDisplayMode,
+      informative,
+      rawValue,
```

Comment:
should `rawValue` be copying over the `rawValue` from the audit result? Should it have a different name if not?

Reply:
looking at usage below, maybe these could have a name that indicates they're overrides